test/test_helper.rb

```diff
@@ -16,8 +16,11 @@
 # <http://www.gnu.org/licenses/agpl.html>
 #
 require 'minitest'
-require 'simplecov'
-SimpleCov.start if (!ENV.has_key?('COV') && !ENV.has_key?('COVERAGE')) || (ENV['COV'] != 'false' && ENV['COVERAGE'] != 'false')
+
+if Rake.application.top_level_tasks.include?('test') && (!ENV.key?('COV') && !ENV.key?('COVERAGE')) || (ENV['COV'] != 'false' && ENV['COVERAGE'] != 'false')
```

Comment:
The formulation of the condition seems odd, I suggest the following
```suggestion
if Rake.application.top_level_tasks.include?('test') && !(ENV.key?('COV') || ENV.key?('COVERAGE')) || (ENV['COV'] == 'true' && ENV['COVERAGE'] == 'true')
```

Reply:
we do not support `COV=true` or `COVERAGE=true`: see our README

Reply:
I guess what Gwen is suggesting, the condition don't read/sound natural. Otherwise, we don't have `COV=false` or `COVERAGE=false` in the README either.

Even after Gwen's modification I think the condition is still too complicated. 

Why don't we do as follows (if our intention is to keep the coverage on by default)

```
if Rake.application.top_level_tasks.include?('test') && [ENV['COVERAGE'], ENV['COV']].exclude? 'false'
```

or if our intention is to let it be disabled by default then we can do 

```
if Rake.application.top_level_tasks.include?('test') && [ENV['COVERAGE'], ENV['COV']].include? 'true'
```

Reply:
What do you think @fab-girard ?